src/test/java/com/google/api/codegen/testdata/no_path_templates_config.baseline

```diff
@@ -1,7 +1,8 @@
 type: com.google.api.codegen.ConfigProto
```

Comment:
Add a top-level comments with the something like:

TODO: Please address all the TODOs in this auto-generated config before using it for client generation. Remove this paragraph after you closed all the TODOs. `retry_codes_name`, `required_fields`, `flattening`, `timeouts`-related config are normally configured by API producers and cannot be preciously decided by the tooling.